lib/api/units/validateBody.js

```diff
@@ -0,0 +1,233 @@
+const jph = require('json-parse-helpfulerror');
+const mediaTyper = require('media-typer');
+const contentTypeUtils = require('content-type');
+
+const { TextDiff } = require('../../../lib/validators/text-diff');
+const { JsonExample } = require('../../../lib/validators/json-example');
+const { JsonSchema } = require('../../../lib/validators/json-schema');
+
+function isPlainText(mediaType) {
+  return mediaType.type === 'text' && mediaType.subtype === 'plain';
+}
+
+function isJson(mediaType) {
```

Comment:
I'm keeping `mediaType` namespace because it's a valid type returned from `mediaTyper.parse()` function. If we used some type superset like TypeScript it would be a little more evident.

Reply:
Using a single `isJson` a6ba96180f7b12e941a47cccfcf0114f8a546c8a